build.gradle

```diff
@@ -242,3 +232,22 @@ if (project.hasProperty("signing.keyId") && project.hasProperty("signing.passwor
     }
 }
 
+// Plugin that uses the New Maven Central Publishing (NMCP) API to publish Maven artifacts. https://github.com/GradleUp/nmcp
+nmcp {
+    centralPortal {
+        username = sonatype_username
+        password = sonatype_password
+        // publish manually from the portal
+        publishingType = "USER_MANAGED"
+//        // or if you want to publish automatically
+//        publishingType = "AUTOMATIC"
```

Comment:
I'm thinking we could set this to `AUTOMATIC` once we see it work well. It saves the step of having to log in to do the publishing, but it also doesn't allow the extra "no I really mean it" phase. I'm open to either way.